### PR TITLE
Fixes and improvements for io, cli and cpwer

### PR DESCRIPTION
**Pull request by boeddeker**

### CODE REVIEW

**Comment by boeddeker** [comment]
Motivation for the name of the `parse_float` argment: `json.loads('1.1', parse_float=decimal.Decimal)`

File: meeteval/wer/__main__.py [hunk 1]
```diff
@@ -444,6 +491,11 @@ def add_command(fn):
                     '--collar', type=positive_number,
                     help='Collar applied to the hypothesis timings'
                 )
+            elif name == 'regex':
```
Comment (thequilo) [inline meeteval/wer/__main__.py:494]: Does it make sense to add the option to all WERs and not just cpWER?

File: meeteval/wer/wer/cp.py [hunk 2]
```diff
@@ -123,15 +124,49 @@ def cp_word_error_rate(
...
+    >>> cp_word_error_rate(r, h)
+    CPErrorRate(errors=0, length=2, insertions=0, deletions=0, substitutions=0, error_rate=0.0, missed_speaker=0, falarm_speaker=0, scored_speaker=1, assignment=(('r0', 'h0'),))
+
+    >>> cp_word_error_rate(['a b c'.split(), 'd e f'.split()], ['a b c'.split(), 'd e f'.split()])
+    CPErrorRate(errors=0, length=6, insertions=0, deletions=0, substitutions=0, error_rate=0.0, missed_speaker=0, falarm_speaker=0, scored_speaker=2, assignment=((0, 0), (1, 1)))
     """
     from meeteval.wer.matching.cy_levenshtein import levenshtein_distance
+    import meeteval.io
+
     def transcription_to_words(x):
```
Comment (thequilo) [inline meeteval/wer/wer/cp.py:143]: We could define a type for this, similar to what was done in the time-constrained WER

Comment (boeddeker) [reply meeteval/wer/wer/cp.py:143]: Maybe later, when we need that somewhere else.

File: meeteval/wer/__main__.py (hunk 1 above)
Comment (boeddeker) [reply meeteval/wer/__main__.py:501]: Done

**Review by thequilo — APPROVED** [review]

**Pull request merged**